### PR TITLE
depracation

### PR DESCRIPTION
**Pull request by maorfr**
this repository is no longer active.

new deployment mechanism is at https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/data/services/launcher/cicd/ci-ext/saas.yaml.

cc @ia3andy @gsmet @gastaldi

### CODE REVIEW

**Comment by gastaldi** [comment]
Is this in place already?

**Comment by maorfr** [comment]
it is.

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Code Quarkus release
 
+## Depracation notice
```
Comment (gastaldi) [inline README.md:3]:
```suggestion
## Deprecation notice
```

**Comment by gastaldi** [comment]
@ia3andy after this is merged can you archive this repository in the GutHub settings? I don't have permission to do that

**Comment by maorfr** [comment]
i just love the "commit changes" on the code suggestions!

**Review by gastaldi — APPROVED** [review]

**Pull request merged**